src/app/forms/templates/test-records/create-master.template.ts

```diff
@@ -32,28 +31,6 @@ const groups1and2Template: Record<string, FormNode> = {
   reasonForCreation: reasonForCreationSection
 }
 
-const group9and10Template: Record<string, FormNode> = {
-  required: CreateRequiredSectionHgvTrl,
-  vehicle: ContingencyVehicleSectionDefaultTrl,
-  test: ContingencyTestSectionGroup9And10,
-  visit: ContingencyVisitSection,
-  notes: NotesSection,
-  defects: DefectsTpl,
-  customDefects: CustomDefectsSection,
-  reasonForCreation: reasonForCreationSection
-}
-
-const group12and14Template: Record<string, FormNode> = {
```

Comment:
These were some changes from my other branch that somehow got mixed in with this commit. 12 and 14 are being worked on a different ticket.